server/views/pages/index.njk

```diff
@@ -31,6 +31,22 @@
             "id": "change-visit"
           }) }}
         </li>
+        {% if showReviewBookingsTile %}
+          <li class="govuk-grid-column-one-third card-group__item">
+            {{ card({
+              "href": "/",
+              "clickable": "true",
+              "heading":
+                '<span class="bapv-nowrap">Need review' +
```

Comment:
Extra `span` elements and `nowrap` class to make sure there's spacing to the left of the badge **and** that it also doesn't wrap - i.e. avoid just. the badge going onto a new line